edx_proctoring/management/commands/set_is_attempt_active.py

```diff
@@ -0,0 +1,102 @@
+"""
+Django management command to update the is_attempt_active field on
+ProctoredExamSoftwareSecureReview and ProctoredExamSoftwareSecureReviewHistory models
+"""
+import logging
+import time
+
+from django.core.management.base import BaseCommand
+
+from edx_proctoring.models import (
+    ProctoredExamSoftwareSecureReview,
+    ProctoredExamSoftwareSecureReviewHistory,
+    ProctoredExamStudentAttempt
+)
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django Management command to update is_attempt_active field on review models
+    """
+    update_field_count = 0
+    update_attempt_codes = []
+    distinct_attempt_codes = set()
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch_size',
+            action='store',
+            dest='batch_size',
+            type=int,
+            default=300,
+            help='Maximum number of attempt_codes to process. '
+                 'This helps avoid locking the database while updating large amount of data.')
+        parser.add_argument(
+            '--sleep_time',
+            action='store',
+            dest='sleep_time',
+            type=int,
+            default=10,
+            help='Sleep time in seconds between update of batches')
+
+    def handle(self, *args, **options):
+        """
+        Management command entry point, simply call into the signal firiing
+        """
+
+        batch_size = options['batch_size']
+        sleep_time = options['sleep_time']
+
+        log.info(u'Updating is_attempt_active field for current reviews.')
```

Comment:
Here and below -- `u` prefix is odd and should be removed since we are on the py3

Reply:
Thanks for these suggestions @idegtiarov, I'll update it!